packages/hmr-plugin/src/utils/externals.ts

```diff
@@ -0,0 +1,7 @@
+import { HRM_RUNTIME } from '../symbols';
+
+declare const window: any;
+
+export function hrmIsReloaded(): boolean {
```

Comment:
`hrm` should be `hmr` 😆 
This spelling mistake is throughout.

Also see the `HRM_RUNTIME` should be `HMR_RUNTIME`

Reply:
done